news-bundle/src/Resources/contao/classes/News.php

```diff
@@ -336,6 +336,11 @@ public function getSearchablePages($arrPages, $intRoot=0, $blnIsSitemap=false)
 				{
 					while ($objArticle->next())
 					{
+						if ($blnIsSitemap && $objArticle->robots === 'noindex,nofollow')
```

Comment:
should this check the page settings as well?

Reply:
It's already checked above, so no need to check it again

https://github.com/contao/contao/blob/449a6fbb3717af62b5f40f1f71cd9f5c71fb87f1/news-bundle/src/Resources/contao/classes/News.php#L320